pkg/apply/prune/prune.go

```diff
@@ -202,6 +206,13 @@ func (p *Pruner) Prune(
 		}
 		taskContext.InventoryManager().AddSuccessfulDelete(id, obj.GetUID())
 		taskContext.SendEvent(eventFactory.CreateSuccessEvent(obj))
+		if opts.MarkAsReconciled {
+			err := taskContext.InventoryManager().SetSuccessfulReconcile(id)
```

Comment:
Hm, I wonder if we should denote somehow that this has been skipped rather than marking it as successfully reconciled. That might allow higher-level libraries/functions/CLIs/whatever to better understand what happened during a run/application?